forwards_paper.tex

```diff
@@ -722,70 +712,13 @@ \subsection*{Simuation methods}
 
 Jaime and Kevin to describe simulation methods here (separately).
 
-
-%%%%%%%%%%%%%%
-\subsection*{Simplification}
-
-Here we describe the simplification algorithm.
-\plr{I am trying it out to see what it looks like.}
-
-\begin{taocpalg}{S}{Simplify a tree sequence}
-{Simplifies the input tree sequence.
-}
-
-
-\algstep{S0.}{Initialize.}{
-    Set the samples.
-}
-
-\algstep{S1.}{Input parent loop.}{Loop over parents, chronologically.
-}
-
-\algstep{S2.}{Process parent edges.}{Loop over edges associated with this parent, removing ancestry from each child.
-}
-
-\algstep{S3.}{Merge ancestors.}{Merge the resulting segments into a new ancestor for this parent.
-}
-
-\end{taocpalg}
-
-This uses
-\begin{taocpalg}{R}{Remove ancestry}
-    {Remove ancestry on a segment from an ancestor.}
-
-
-\end{taocpalg}
-and
-\begin{taocpalg}{M}{Merge ancestors}
-    {Merge a collection of segments and create a new ancestor.}
-
-
-\end{taocpalg}
-
-
 %%%%%%%%%%%%%%
 \subsection*{Data structures:}
```

Comment:
Do we want something like this here?  The purpose would be to lay out some more of the properties and requirements of the tables; explaining why sort_tables and simplify are necessary.  That certainly belongs in the documentation, but maybe it's still useful to talk about more verbosely here.

But, I'm guessing we *don't* want this here?

Reply:
I'd vote for leaving it out anyway. Seems like technical details more suited to documentation.

Reply:
sounds good.  this is mostly addressed by the API section now.

well, except I kinda want to include the list of requirements, at least the ones that ensure the tables make sense, since that's a theoretical result.  we can drop that bit later.